diff --git a/node_modules/react-native-svg/src/index.d.ts b/node_modules/react-native-svg/src/index.d.ts
index 891a99d..e8de3c6 100644
--- a/node_modules/react-native-svg/src/index.d.ts
+++ b/node_modules/react-native-svg/src/index.d.ts
@@ -264,6 +264,7 @@ export type Ellipse = React.ComponentClass<EllipseProps>;
 
 export interface GProps extends CommonPathProps, FontProps {
   opacity?: NumberProp;
+  children?: React.ReactNode;
 }
 export const G: React.ComponentClass<GProps>;
 export type G = React.ComponentClass<GProps>;
